src/api/client.go

```diff
@@ -256,7 +328,7 @@ func (c *Client) PostJSONV2(endpoint string, reqObj, respObj interface{}) (bool,
 
 // CSRF returns a CSRF token. If CSRF is disabled on the node, returns an empty string and nil error.
 func (c *Client) CSRF() (string, error) {
-	resp, err := c.get("/api/v1/csrf")
+	resp, err := c.get("/api/v1/csrf", "")
```

Comment:
why this?

Reply:
Because, I generalised the get-method in the client, for less redundant code.